### PR TITLE
Use primary keys info while generating incremental queries

### PR DESCRIPTION
**Pull request by ypc-faros**
## Description

Use primary keys info while generating incremental queries

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/faros-graphql-source/package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "faros-airbyte-cdk": "0.0.1",
     "faros-airbyte-common": "0.0.1",
-    "faros-js-client": "0.1.6",
+    "faros-js-client": "^1.0.0",
```
Comment (tovbinm) [inline sources/faros-graphql-source/package.json:34]: When did we bump to 1.0.0 ?!

**Review by tovbinm — COMMENTED** [review]
Why did so many lines change in package lock file?

**Comment by ypc-faros** [comment]
> Why did so many lines change in package lock file?

Dealing with that + merge conflicts at the moment. Please ignore for now

File: sources/faros-graphql-source/package.json (hunk 1 above)
Comment (ypc-faros) [reply sources/faros-graphql-source/package.json:34]: There was a breaking change introduced in https://github.com/faros-ai/faros-js-client/pull/30

**Comment by ypc-faros** [comment]
Dropped in favor of https://github.com/faros-ai/airbyte-connectors/pull/802
CC @tovbinm

**Pull request closed without merging**